### PR TITLE
Reverted api to allowing multiple options per condition

### PR DESCRIPTION
**Pull request by SamGodwin2**
What is the context of this PR?
This PR has been raised in order to revert some the changes made to routing and return it to being able to select multiple options per condition.

How to review
Tests pass, changes make sense.

### CODE REVIEW

**Comment by hamishtaplin** [comment]
screenshot 2018-07-02 16 43 59

File: docker-compose.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ services:
       - db
     volumes:
       - .:/app
-      # - ./node_modules/eq-author-graphql-schema:/app/node_modules/eq-author-graphql-schema
+      - ./node_modules/eq-author-graphql-schema:/app/node_modules/eq-author-graphql-schema
```
Comment (samiwel) [inline docker-compose.yml]: Be sure to comment this back out once we're done.

**Comment by SamGodwin2** [comment]
@hamishtaplin Found that error yesterday as well but forgot to push the fix up, should all be working now though.

**Comment by samiwel** [comment]
Looks good, I just wonder if it's worth pinning to the correct version of the schema given the breaking change between v0.15.0 and v0.15.1?

e.g. `yarn upgrade eq-author-graphql-schema@0.15.1`

**Pull request merged**